src/kili/mutations/plugins/__init__.py

```diff
@@ -0,0 +1,128 @@
+"""Project mutations."""
+
+
+from typing import Optional
+
+from typeguard import typechecked
+
+from kili.services.plugins import (
+    PluginUploader,
+    activate_plugin,
+    deactivate_plugin,
+    delete_plugin,
+)
+
+from ...authentication import KiliAuth
+
+
+class MutationsPlugins:
+    """Set of Plugins mutations."""
+
+    # pylint: disable=too-many-arguments,too-many-locals
+
+    def __init__(self, auth: KiliAuth):
+        """Initialize the subclass.
+
+        Args:
+            auth: KiliAuth object
+        """
+        self.auth = auth
+
+    @typechecked
+    def upload_plugin_beta(
+        self,
+        file_path: str,
+        plugin_name: Optional[str] = None,
+        verbose: bool = True,
+    ):
```

Comment:
quick UX question: shouldn't be useful for the client to have its script automatically activated when uploading it ? And this could be disable with an bool argument like disable_auto_activation ?

Reply:
Nope because a plugin lives on its own. Then a plugin can be activated on one or several projects. When you create it, it's not linked to any project (=not activated on any project)